docs/walkthrough/create-training-data.md

```diff
@@ -68,13 +84,22 @@ This is another English sentence    Dies ist ein weiterer englischer Satz
 ```
 
 ```{warning}
-When reading in files constructed in this way, a unique label is generated for each pair. Some sampling methods that require more than two elements per class will not work because each class in the dataset will only contain a single pair of elements.
-
-
+When reading in files constructed in this way, a unique label is generated for each pair.
+Some sampling methods that require more than two elements per class will not work because each class in the dataset will only contain a single pair of elements.
 ```
 
 ````
 
+It is worth noting that we support the following dialects:
+
++ `excel` use `,` as delimiter and `\r\n` as lineterminator.
++ `excel-tab` use `\t` as delimiter and `\r\n` as lineterminator.
++ `unix` use `,` as delimiter and `\n` as lineterminator.
+
+```{warning}
+Please remove/replace comma in your data fields if you are using a comma `,` as a delimiter.
```

Comment:
Does the quoting convention in CSV work in this importer?

"foo","bar","foo, bar"\n

Reply:
if i understand you correctly, i think it will be imported as it is, including the quoting in column strings.